c2pa-proposal.md

```diff
@@ -98,3 +98,104 @@ The only requirement for the Implicit Attestation to work within C2PA Eco-system
     4) If the signing key is revoked for any reason (e.g. due to expiry), then a new key to be provisioned, has to undergo the same "Explicit Attestation Protocol".
 
 Specific Explicit "Key Attestation" Protocol is out of scope of C2PA Specification.
+
+## Explicit Attestations
+
+Explicit Attestation allow a C2PA Claim Signer to undergo attestation with a local or remote
```

Comment:
Do we want to allow an explicit attestation to be a "raw" SGX or TPM quote? I think so, which is why you use the term "local trust broker" but I think that this will be confusing for people. Suggest:
"Explicit attestations directly incorporate evidence of a platform's security configuration in the manifest. Explicit attestations can be "raw" evidence obtained from a local security component (e.g. an SGX or TPM quote) or can be an integrity verdict from a third-party trust broker such as those defined by IETF RATS or Google Play Integrity.

Reply:
@paulengland I agree with your comments and I will incorporate the same into the text.